src/core_modules/capture-core/components/WidgetEventEdit/EventChangelogWrapper/EventChangelogWrapper.component.js

```diff
@@ -42,9 +42,19 @@ export const EventChangelogWrapper = ({ formFoundation, eventId, eventData, ...p
             return acc;
         }, {});
 
+        const additionalFields = formFoundation.featureType !== 'None' ? {
+            geometry: {
+                id: 'geometry',
+                name: formFoundation.featureType,
```

Comment:
Can we use the same labels as the ones currently displayed in the form? Something like this:
`formFoundation.featureType === 'Polygon' ? i18n.t('Area'): i18n.t('Coordinate')`